src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/RolesServiceTest.kt

```diff
@@ -50,7 +50,16 @@ class RolesServiceTest {
     fun `get all paged roles`() {
       val roles = createRolePaged()
 
-      whenever(externalUsersService.getPagedRoles(anyInt(), anyInt(), anyString(), isNull(), isNull(), isNull())).thenReturn(roles)
+      whenever(
+        externalUsersService.getPagedRoles(
+          anyInt(),
+          anyInt(),
+          anyString(),
+          isNull(),
+          isNull(),
+          isNull()
+        )
+      ).thenReturn(roles)
```

Comment:
Are you using a different ktLint to everybody else cos I'm sure mine didn't pick this up?  very odd

Reply:
The was a couple of Assertions that weren't imported so ran `./gradlew ktlintFormat`